src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -66,7 +66,7 @@
     --metric-unit-color: #999;
     --metric-label-background: #777;
     --metric-label-color: #e0e0e0;
-    --trace-selected-background: #71663a;
+    --trace-selected-background: #71663acc;
```

Comment:
The `#RRGGBBAA` notation for colors is not supported by Edge. I suggest using the `rgba()` notation instead.